src/v3/src/util/webauthnUtils.ts

```diff
@@ -82,10 +82,10 @@ export const webAuthNAuthenticationHandler: WebAuthNAuthenticationHandler = asyn
   // Generates a CredentialRequestOptions Object for the Web Auth API to
   // generate a PublicKeyCredential instance for use by IDX verify the user
   const options = (
-    challengeData && authenticatorEnrollments?.value
+    challengeData
   ) && OktaAuth.webauthn.buildCredentialRequestOptions(
     challengeData,
-    authenticatorEnrollments.value,
+    authenticatorEnrollments?.value ?? [],
```

Comment:
Same approach is used in Gen2:
https://github.com/okta/okta-signin-widget/blob/9f6dea0441635b16cddc18d5bb8f20c7d97010e6/src/v2/view-builder/views/webauthn/ChallengeWebauthnView.js#L77
Without this fix `identify-with-webauthn-launch-authenticator.json` mock will produce error `Cannot read properties of null (reading 'response')`

Reply:
Are we sure that the mock itself is accurate? That would mean that in real usage this was throwing the error as well.

Reply:
It seems not accurate.
In reality I see that if `authenticatorEnrollments` is empty or not contains webauthn enrollment, then webauthn authenticator is not available for challenging

Reply:
Reverted the code change. Added exception for en leaks and ref to OKTA-654489
https://github.com/okta/okta-signin-widget/pull/3714/files#diff-20ecefc4d164e1f22564fa444b02083080727de4a7c205be486bfbacbbbd050dR41